_assets/ci/Jenkinsfile

```diff
@@ -24,55 +24,109 @@ node('linux') {
     gitShortSHA = gitSHA.take(7)
     gitBranch = sh(returnStdout: true, script: 'git name-rev --name-only HEAD').trim() - remoteOriginRegex
 
-    stage('Debug') {
+    // For branch builds, replace the old artifact. For develop keep all of them.
+    version = gitBranch == 'develop' ? getVersion(gitBranch, gitShortSHA, '') : getVersion(gitBranch, gitShortSHA, env.BUILD_ID)
```

Comment:
I recently changed the way the version is calculated. Check out `Jenkinsfile-manual`:
```
def version = sh(
                script: "git describe --exact-match --tag 2>/dev/null || git describe --always",
                returnStdout: true
            ).trim()
```